int32 id
string type
geometry_msgs/Pose pose
---

